src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```

Comment:
- [ ] according to the requirements you should make use of class components instead of functional components. please change the `App` component into a class component.